modules/utxo-lib/src/bitgo/wallet/Psbt.ts

```diff
@@ -15,35 +21,76 @@ import { isWalletUnspent, WalletUnspent } from './Unspent';
 import { checkForInput } from 'bip174/src/lib/utils';
 import { PsbtInput } from 'bip174/src/lib/interfaces';
 import {
-  getScriptPathLevel,
+  getLeafVersion,
   calculateScriptPathLevel,
   isValidControlBock,
-  ParsedPubScript2Of3,
+  ParsedPubScriptP2ms,
   ParsedPubScriptTaprootScriptPath,
-  parsePubScript,
+  parsePubScript2Of3,
+  ParsedPubScriptTaproot,
+  ParsedPubScriptTaprootKeyPath,
 } from '../parseInput';
+import { parsePsbtMusig2PartialSigs } from '../Musig2';
+import { isTuple } from '../types';
 
-type Signatures = [Buffer] | [Buffer, Buffer] | undefined;
+type SignatureType = [Buffer] | [Buffer, Buffer] | undefined;
```

Comment:
not sure why the `Type` suffix here was added - unlike the classification strings, this really still describes the signature data